client/components/payment-methods-list/index.js

```diff
@@ -0,0 +1,21 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React from 'react';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+const PaymentMethodsList = ( { className, children } ) => {
```

Comment:
This used to be called `OrderableList` - I just renamed it to `PaymentMethodsList` and removed the `@dnd-kit` library dependencies.
Unfortunately it is being picked up as a new file.